go.mod

```diff
@@ -106,4 +106,6 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
+replace github.com/codeready-toolchain/api => github.com/mfrancisc/api v0.0.0-20221108133602-c5b3a69211db
```

Comment:
friendly reminder

Reply:
fixed in [dbd114](https://github.com/codeready-toolchain/host-operator/pull/701/commits/dbd114c716918cec482d6b108526a3164b9fbc5c)